src/commands/commands.js

```diff
@@ -131,6 +131,60 @@ async function handler (error, callback = () => Promise.resolve()) {
   }
 }
 
+async function getPlaybackInfo () {
+  return await axios.get(SPOTIFY_PLAYER_URL)
+}
+
+async function putVolume(value) {
+  try {
+    await axios
+      .put(`${SPOTIFY_PLAYER_URL}/volume?volume_percent=${value}`)
+    window.setStatusBarMessage(`Volume: ${value}%`, 2000)
+  } catch (error) {
+    console.error(error)
+  }
+}
+
+async function decreaseVolume () {
+  const { data: currentPlayback } = await getPlaybackInfo()
+  if (!currentPlayback) return;
+  const decreasedVolume = currentPlayback.device.volume_percent < 10
+    ? 0 : currentPlayback.device.volume_percent - 10
+  putVolume(decreasedVolume)
+}
+
+async function increaseVolume () {
+  const { data: currentPlayback } = await getPlaybackInfo()
+  if (!currentPlayback) return;
+  const increasedVolume = currentPlayback.device.volume_percent > 90
+    ? 100 : currentPlayback.device.volume_percent + 10
+  putVolume(increasedVolume)
+}
+
+async function getUserPlaylists () {
+  const { data: userPlaylistsData } = await axios.get(`${USER_PLAYLISTS_URL}`, {})
+  const basicPlaylistsData = userPlaylistsData.items.map(playlist =>
+    Object.assign({
```

Comment:
why are you  using `Object.assign`? Do you know that can you do just this ? 
```
const basicPlaylistsData = userPlaylistsData.items.map(playlist => {description: `Owner: ${playlist.owner.display_name}`, details: playlist.uri,label: playlist.name});
```

Reply:
I remember now that I actually tried to return using just `=> { ... }`, but I was getting an error. What I should've done was returning like this: `=> ({ ... })`. hehehe